src/elements/content-sidebar/stories/tests/MetadataSidebarRedesign-visual.stories.tsx

```diff
@@ -1,12 +1,21 @@
 import { type ComponentProps } from 'react';
 import { expect, userEvent, within, fn, screen } from '@storybook/test';
 import { type StoryObj } from '@storybook/react';
+import { http, HttpResponse } from 'msw';
 import { defaultVisualConfig } from '../../../../utils/storybook';
 import ContentSidebar from '../../ContentSidebar';
 import MetadataSidebarRedesign from '../../MetadataSidebarRedesign';
-
-const fileIdWithMetadata = global.FILE_ID;
-const fileWithoutMetadata = '416047501580';
+import {
+    fileId,
+    fileIdWithoutMetadata,
+    mockEmptyMetadataInstances,
+    mockEnterpriseMetadataTemplates,
+    mockFileRequest,
+    mockFileRequestWithoutMetadata,
+    mockMetadataInstances,
+} from '../__mocks__/MetadataSidebarRedesignedMocks';
+
+const fileIdWithMetadata = fileId;
```

Comment:
Can't it be named using `fileIdWithMetadata` when it's defined?

Reply:
Wiola is off till Nov 13, and I don't have access to her fork, so I think we can rename it in future PRs.